scripts/rv_expression_association/run_SKAT.R

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(suppressMessages(install.packages("googleCloudStorageR", repos = 'http://cran.csiro.au', quietly = T)))
```

Comment:
Just curious if you actually need these suppression messages, or if it just makes the logs look cleaner? If it's important, maybe add a general comment about why you need it

Reply:
no i don't at all (it's just aesthetics), and in fact this didn't even really work! I might drop them for now and if I do find a working solution add a comment to clarify this is just to make the log cleaner